buildscripts/buildRelease.sh

```diff
@@ -264,40 +266,38 @@ git add $NextVersion/ assets/ docs/ modules/ showcase/ *.css *.html *.ico *.js
 git commit -a -m "Committing documentation changes for $NextVersion-gh-pages-test. Latest commit ID is $latestCommitID."
 git push origin $NextVersion-gh-pages-test
 
-# Return to the develop branch and discard changes to a couple of files
-echo
-echo Returning to the develop branch
-cd $WORKSPACE
-git checkout docs/app/data/footer-navigation.json
-git checkout docs/app/data/landing-page.json
-
-# Create the new branch for the Keppel bower package
+# Create a branch for the new Keppel Bower package. First, clone the repository to a sub-folder.
+# Switching to the new branch and commiting to it will be performed in this clone.
 echo
 echo Creating the branch $NextVersion-package-test
+cd $WORKSPACE
+mkdir package-clone
+cd package-clone
+git clone git@github.com:UXAspects/UXAspects.git
+cd UXAspects
+git checkout bower
```

Comment:
I believe you can safely cut down these lines and make this more efficient, try:

git clone -b bower --single-branch git@github.com:UXAspects/UXAspects.git